pages/index.tsx

```diff
@@ -1,12 +1,56 @@
 // pages/index.tsx
 import { Guider } from '../components';
+import { Target } from '../components'
 
-const Home = () => (
-  <div>
-    <h1>Telescope Control</h1>
-    <Guider />
-  </div>
-);
+const starList = `
```

Comment:
Maybe the sample data could be stored in a separate file and imported from there?